splitio/push/splitworker.py

```diff
@@ -63,16 +64,17 @@ def _run(self):
                 continue
             _LOGGER.debug('Processing feature flag update %d', event.change_number)
             try:
-                if event.compression is not None and event.previous_change_number == self._feature_flag_storage.get_change_number():
-                    try:
-                        self._feature_flag_storage.put(from_raw(json.loads(self._get_feature_flag_definition(event))))
-                        self._feature_flag_storage.set_change_number(event.change_number)
-                        continue
-                    except Exception as e:
-                        _LOGGER.error('Exception raised in updating feature flag')
-                        _LOGGER.debug(str(e))
-                        _LOGGER.debug('Exception information: ', exc_info=True)
-                        pass
+                if event.update_type == UpdateType.SPLIT_UPDATE:
```

Comment:
why we need to check this type of event? I believe that those are the only events that are queued by the processor. If we need this check, I would suggest having the opposite and `pass` to avoid multiple if conditions

```
if event.update_type != UpdateType.SPLIT_UPDATE:
   do nothing
// rest of the code